File: Tutorials/ASP.NET/MVC5/CS/Models/CustomerViewModel.cs
using System.Collections.Generic;

namespace AspNetMvcApplication.Models {
    public class CustomerViewModel {
        public int Oid { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string ContactName {
            get { return string.Concat(FirstName, " ", LastName); }
        }
        public List<OrderViewModel> Orders { get; set; }
    }
}

File: Tutorials/ASP.NET/MVC5/VB/Models/CustomerViewModel.vb
Imports System.Collections.Generic

Namespace AspNetMvcApplication.Models
	Public Class CustomerViewModel
		Public Property Oid() As Integer
		Public Property FirstName() As String
		Public Property LastName() As String
		Public ReadOnly Property ContactName() As String
			Get
				Return String.Concat(FirstName, " ", LastName)
			End Get
		End Property
		Public Property Orders() As List(Of OrderViewModel)
	End Class
End Namespace
